docs/user/quickstart.rst

```diff
@@ -211,6 +211,7 @@ Note: Custom headers are given less precedence than more specific sources of inf
 
 Furthermore, Requests does not change its behavior at all based on which custom headers are specified. The headers are simply passed on into the final request.
 
+Note: All header values must be a ``string``, bytestring (on Python 2), or unicode (on Python 3).
```

Comment:
Tweaked the wording to avoid version listing, and added a suggestion to avoid passing unicode values in both places.